src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/rectangleSelection.scss

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
I think that comment moved for some reason :/ At the beginning the new line was at the very beginning... Or maybe I am mixing something up.